frontend/src/components/managePermissions/table/DataTableHeader.vue

```diff
@@ -12,33 +13,37 @@ const props = defineProps({
     },
     btnRoute: {
         type: String,
-        required: true
+        required: true,
     },
     filter: {
         type: String,
-        required: true
-    }
+        required: true,
+    },
 });
 const emit = defineEmits(['change']);
 
 const computedFilter = computed({
     get() {
-        return props.filter
+        return props.filter;
     },
     set(newValue) {
-        emit('change', newValue)
-    }
-})
+        emit('change', newValue);
+    },
+});
 
+const userLevelText = computed(() => {
+    if (props.btnRoute == routeItems.grantDelegatedAdmin.path)
```

Comment:
I was thinking if pass another property or just check the button route to determine if this is for delegated admin table or regular table, feel both okay. And not sure if this naming makes sense, I'm bad at naming...